desktop/cmp/panel/PanelSizingModel.js

```diff
@@ -28,6 +27,9 @@ export class PanelSizingModel {
     side;
     collapsedRenderMode;
     prefName;
+    showSplitter;
```

Comment:
I think splitter is probably the right thing to call this.  Could alternatively call it 'resizeBar'.  

One simplification we could do, is just always show the splitter collapse button if the splitter was being shown, and the panel was collapsible.  Or do you think we might want to *not* show it, if we are already showing it in the header?  I think this is all probably right.

Reply:
Also, one thing to consider is moving these to panel itself?  I think you *could* argue that they might go there?  Not 100% sure about this, and I do realize that 'side' is here on the model

(also realize that implementation-wise, a lot easier to have these on the model)

Reply:
I had it as `resizeBar` or something similar originally, but realized it actually has nothing to do with resizing so I went with splitter. 

Yes was thinking you might not want the splitter button shown if you have the header button, would probably do that for my client app use-case. Don't feel too strongly about it though so happy to make a change if we want to eliminate a prop to simplify the api.

Also went back and forth on where these props should live, and ultimately decided that it seemed weird for them to be on the panel when the other props like `side` and `collapsible` which control if/how these things are rendered are on the model. Could see us moving the non-state props (`resizable`, `collapsible`, `side`, `collapsedRenderMode`) and these new props to the Panel component and limiting the PanelSizingModel to just maintaining the collapse state and size of the panel, though as you pointed out having everything on the model makes the implementation nice and clean for the child components.